docs/markdown/03_langages/00-TITLE.md

```diff
@@ -639,6 +639,79 @@ resource "google_compute_instance" "web" {
 }
 ```
 
+##==##
+<!-- .slide: class="with-code-bg-dark"-->
+
+# HCL extended (also known as HIL)
+
+<br/>
+
+## Dynamic blocks
+
+Certaines resources acceptent des blocs pour leurs configurations. Par exemple, le life cycle d'un bucket GCS ressemblera à ceci :
+
+```hcl-terraform
+resource "google_storage_bucket" "auto-expire" {
+  name          = "auto-expiring-bucket"
+  location      = "EU"
+
+  lifecycle_rule {
+    condition {
+      age = 3
+    }
+    action {
+      type = "Delete"
+    }
+  }
+}
+```
+
+Ce "lifecycle_rule" n'est pas pas un simple attribut qui accepte un map, mais c'est un "bloc".
+
+Généralement, ces blocs peuvent être présent plusieurs fois dans une même resource (pour définir plusieurs life_cycle_rule dans l'example).
+
+
+##==##
+<!-- .slide: class="with-code-bg-dark"-->
+
+# HCL extended (also known as HIL)
+
+<br/>
+
+## Dynamic blocks
+
+Pour générer dynamiquement ces blocs, vous aurez besoin d'écrire des "[Dynamic blocks](https://www.terraform.io/language/expressions/dynamic-blocks)" :
+
```

Comment:
Il faudrait ajouter le meta-param `iterator` et mettre en évidence que l'on doit utiliser soit le "nom" de l'attribut, soit la valeur de l'itérateur pour récupérer la valeur (et non pas each.xxx)

Reply:
+1, pour le reste, LGTM

Reply:
J'ai ajouté un blabla sur iterator